File: price.php
#!/usr/bin/env php
<?php
function get_title($html_page)
{
  // split the page into 3 sections, with the <title> and </title> tags as delimiters.
  $split_page = preg_split("%</?title[^>]*>%",$html_page);
  return $split_page[1];
}
function get_bit_coin_price()
{
  $bitcoin_price_address = "http://www.bitcoinexchangerate.org/price";
  $html = fread(fopen($bitcoin_price_address, "r"), 100);
  return get_title($html);
}
$price = trim(get_bit_coin_price());
echo $price;
?>

File: price.rb
#!/usr/bin/env ruby

#install nokogiri using gem install nokogiri
require 'rubygems'
require 'nokogiri'
require 'open-uri'
price_page = Nokogiri::HTML open('http://www.bitcoinexchangerate.org/price')
price =  price_page.title.strip.to_f
puts price
